app/views/accounts/new.html.erb

```diff
@@ -0,0 +1,5 @@
+<section class="container">
+  <h1>New Transaction</h1>
+  <%= link_to '< Back', group_accounts_path %>
+  <%= render "form", account: @account %>
```

Comment:
[REQUIRED] Kindly ensure that the user is redirected to the transactions page after successful submission. This seems to be the better UX